pkg/graph/resolvers/plan_discussion.go

```diff
@@ -317,13 +362,87 @@ func sendPlanDiscussionTaggedSolutionEmail(
 	return nil
 }
 
+func GetEmailRecipients(
```

Comment:
Since this is scoped to the entire resolver package, I think that it should probably have a more specific name. Can you please also add a function description it to describe the use of the function?